common/sgx/verifier.c

```diff
@@ -543,11 +543,11 @@ oe_result_t oe_sgx_extract_claims(
 
 done:
     if (claims)
-        sgx_attestation_plugin_free_claims_list(NULL, claims, claims_length);
+        oe_sgx_free_claims_list(NULL, claims, claims_length);
     return result;
 }
 
-static oe_result_t _verify_evidence(
+oe_result_t os_sgx_verify_evidence(
```

Comment:
Typo? Is the intention "oe_sgx_verify_evidence"? Need to check naming convention. Maybe it should be "sgx_attestation_plugin_verify_evidence".

Reply:
Yeah, that's a typo. I think the name is fine, but I'm also happy to rename this and other functions if we should apply a different convention.